packages/dm-core-plugins/blueprints/generic-list/GenericTablePluginConfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "CORE:Blueprint",
+  "name": "GenericTablePluginConfig",
+  "attributes": [
+    {
+      "attributeType": "string",
+      "type": "CORE:BlueprintAttribute",
+      "description": "Name of primitive child attributes to display in header of item",
+      "name": "columns",
+      "default": ["name", "type"],
```

Comment:
These are just settings for which columns to show. There is no requirements on the entity to actually have this data.
Though this would be a useful default, and be able to test for handling "null" values.

Reply:
It is optional. The components default are defined in "GenericTablePlugin.tsx:L26".
Ask me if it's still unclear

Reply:
It's true that GenericTablePlugin says it is optional, but this file says that their mandatory. Shouldn't they align?

Reply:
I misunderstood. All attributes are now "optional" in the blueprint :sweat_smile: